openquake/calculators/postproc/compute_rtgm.py

```diff
@@ -827,17 +826,19 @@ def main(dstore, csm):
         [sids] = locs.values()
         for sid in sids:
             sid_notifications = notifications[notifications['sid'] == sid]
-            if len(sid_notifications) ==0:
+            if len(sid_notifications) == 0:
                 plot_mean_hcurves_rtgm(dstore, sid, update_dstore=True)
                 plot_governing_mce(dstore, sid, update_dstore=True)
                 plot_disagg_by_src(dstore, sid, update_dstore=True)
-            elif sid_notifications['name'][0] not in [
+            elif sid_notifications['name'][0 ] not in [
```

Comment:
Am I understanding correctly that for now, until we have a proper test, we are still not running plot_disagg_by_src in this case?